homeassistant/components/sonos/media_player.py

```diff
@@ -236,6 +236,7 @@ async def async_service_handle(service_call: ServiceCall) -> None:
             vol.Optional(ATTR_NIGHT_SOUND): cv.boolean,
             vol.Optional(ATTR_SPEECH_ENHANCE): cv.boolean,
             vol.Optional(ATTR_STATUS_LIGHT): cv.boolean,
+            vol.Optional(ATTR_EQ_BASS): vol.Coerce(int),
```

Comment:
@pvizeli Would this be better as `vol.Optional(ATTR_EQ_BASS): vol.All(vol.Coerce(int), vol.Range(-10, 10)),`?